torch/_inductor/simple_fsdp.py

```diff
@@ -83,29 +82,25 @@ def reorder_reduce_scatter(
     result_list: List[scheduler.BaseSchedulerNode] = []
     wait_list: List[scheduler.BaseSchedulerNode] = []
     node_to_type: Dict[scheduler.BaseSchedulerNode, int] = {}
-
+    inverse_users, node_users = comms.compute_node_users(snodes)
     prev_node_type: List[int] = []
```

Comment:
can be removed

Reply:
we introduce this line here bc we need to get wait's dependency node here. (I'll fix other things): https://github.com/ruisizhang123/pytorch_intern24/blob/8b7b39626a6b7faf8abe5c337c65ae982ea8c354/torch/_inductor/simple_fsdp.py#L100